database/factories/UserFactory.php

```diff
@@ -25,6 +37,35 @@
         'ends_at' => $faker->dateTimeBetween('+11 days', '+20 days'),
         'cfp_starts_at' => $faker->dateTimeBetween('-9 days', '-1 day'),
         'cfp_ends_at' => $faker->dateTimeBetween('+1 days', '+2 days'),
+        'approved' => false,
+    ];
+});
+
+$factory->state(App\Conference::class, 'closedCFP', function (Faker $faker) {
+    return [
+        'author_id' => function () {
+            return factory(App\User::class)->create()->id;
+        },
+        'title' => 'Dummy Conference',
+        'description' => $faker->sentence,
+        'url' => $faker->domainName,
+        'starts_at' => $faker->dateTimeBetween('+3 days', '+10 days'),
+        'ends_at' => $faker->dateTimeBetween('+11 days', '+20 days'),
+        'cfp_starts_at' => $faker->dateTimeBetween('-9 days', '-1 day'),
+        'cfp_ends_at' => $faker->dateTimeBetween('-3 days', '+2 days'),
```

Comment:
This can end up with a CFP that starts 1 day ago and ends 3 days ago, which isn't possible. Could you update the cfp_starts at max to be 4 days ago?